jedi/inference/compiled/access.py

```diff
@@ -498,7 +493,7 @@ def get_signature_params(self):
                 default_string=repr(p.default),
                 has_annotation=p.annotation is not p.empty,
                 annotation=self._create_access_path(p.annotation),
-                annotation_string=self._annotation_to_str(p.annotation),
+                annotation_string=inspect.formatannotation(p.annotation),
```

Comment:
Where does this function come from? I can't find anything in the documentation.

Reply:
Please return an empty string for Python 2. I don't want this to break in that case.

> I can't find anything in the documentation.

Yeah, I'm also a bit worried by that. Especially because `inspect.formatargspec` (the function that uses it) is deprecated. However Python is usually very stable about functions that have no leading underscore, especially for micro versions. I think it's fine to use this.

Also note that I'm not a big fan of the function at all. It's buggy (it does just replace `typing.`) and not well maintained. I just think that it's not going to matter for most people.